server/server.py

```diff
@@ -108,6 +108,24 @@ def get(self):
         model_card_dict = mc_reconstructor.get_all_mcs()
         return model_card_dict, 200
 
+# Get deployment information
+@api.route('/deployments')
```

Comment:
This should be called something like model_deployments or get_model_deployments. There are deployments for both models and experiments (and from devices). We should distinguish that somehow.